website/server/models/user/hooks.js

```diff
@@ -134,7 +134,7 @@ function _setProfileName (user) {
 schema.pre('validate', function preValidateUser (next) {
   // Populate new user with profile name, not running in pre('save') because the field
   // is required and validation fails if it doesn't exists like for new users
-  if (this.isNew) {
+  if (this.isNew && !this.profile.name) {
```

Comment:
Right now when a new user is created it cannot have a profile name, it's set automatically to the username or google/fb name. I've only added ` && !this.profile.name` because a lot of tests relied on creating a user with a profile name already set but it's not a behavior that we have on the website